swift_browser_ui/_convenience.py

```diff
@@ -318,29 +270,28 @@ def get_availability_from_token(
 
     # Check projects from the API
     prq = urllib.request.Request(
-        str(setd['auth_endpoint_url']) + "/OS-FEDERATION/projects",
+        str(setd["auth_endpoint_url"]) + "/OS-FEDERATION/projects",
         headers=hdr,
     )
     with urllib.request.urlopen(prq, timeout=20) as projects:  # nosec
-        output_projects = json.loads(projects.read().decode('utf-8'))
+        output_projects = json.loads(projects.read().decode("utf-8"))
 
     # Check domains from the API
     drq = urllib.request.Request(
-        str(setd['auth_endpoint_url']) + "/OS-FEDERATION/domains",
+        str(setd["auth_endpoint_url"]) + "/OS-FEDERATION/domains",
         headers=hdr,
     )
     with urllib.request.urlopen(drq, timeout=20) as domains:  # nosec
-        output_domains = json.loads(domains.read().decode('utf-8'))
+        output_domains = json.loads(domains.read().decode("utf-8"))
 
-    logging.info("%s\n%s",
-                 str(output_projects),
-                 str(output_domains))
+    logging.info("%s\n%s", str(output_projects), str(output_domains))
```

Comment:
```suggestion
    logging.info(f"{str(output_projects)}\n{str(output_domains)}")
```

Reply:
bundled these suggestions to a single commit